tests/api/test_checkout_bench.py

```diff
@@ -0,0 +1,602 @@
+import pytest
+from django.conf import settings
+from graphene import Node
+
+from saleor.checkout.utils import add_variant_to_checkout
+from saleor.payment import ChargeStatus, TransactionKind
+from saleor.payment.models import Payment
+from tests.api.utils import get_graphql_content
+
+
+@pytest.fixture()
+def checkout_with_variant(checkout, variant):
+    add_variant_to_checkout(checkout, variant, 1)
+    checkout.save()
+    return checkout
+
+
+@pytest.fixture()
+def checkout_with_shipping_method(checkout_with_variant, shipping_method):
+    checkout = checkout_with_variant
+
+    checkout.shipping_method = shipping_method
+    checkout.save()
+
+    return checkout
+
+
+@pytest.fixture()
+def checkout_with_billing_address(checkout_with_shipping_method, address):
+    checkout = checkout_with_shipping_method
+
+    checkout.billing_address = address
+    checkout.save()
+
+    return checkout
+
+
+@pytest.fixture()
+def checkout_with_charged_payment(checkout_with_billing_address):
+    checkout = checkout_with_billing_address
+
+    payment = Payment.objects.create(
+        gateway=settings.DUMMY,
+        is_active=True,
+        total=checkout.get_total().gross.amount,
+        currency="USD",
+    )
+
+    payment.charge_status = ChargeStatus.FULLY_CHARGED
+    payment.captured_amount = payment.total
+    payment.checkout = checkout_with_billing_address
+    payment.save()
+
+    payment.transactions.create(
+        amount=payment.total,
+        kind=TransactionKind.CAPTURE,
+        gateway_response={},
+        is_success=True,
+    )
+
+    return checkout
+
+
+@pytest.mark.django_db
+@pytest.mark.count_queries
+def test_create_checkout(api_client, graphql_address_data, variant):
+    query = """
+        fragment Price on TaxedMoney {
+          gross {
+            amount
+            localized
+            __typename
```

Comment:
We can safely drop all `__typename` occurrences - they're used by Apollo internally but don't impact the database performance.